src/gradient/_base_client.py

```diff
@@ -376,6 +376,8 @@ def __init__(
         timeout: float | Timeout | None = DEFAULT_TIMEOUT,
         custom_headers: Mapping[str, str] | None = None,
         custom_query: Mapping[str, object] | None = None,
+        user_agent: str | None = None,
```

Comment:
Maybe rename this `user_agent_package` or something as `user_agent` sets the expectation you can override the whole string.